dds/idl/langmap_generator.cpp

```diff
@@ -1379,6 +1387,27 @@ struct Cxx11Generator : GeneratorBase
       "using " << nm << " = " << array << elem_type << bounds.str() << ";\n";
   }
 
+  void gen_anonymous_type(FieldInfo& af)
```

Comment:
This function share the same basic elements with `gen_array` and `gen_sequence`. Maybe create a `gen_array_i` and `gen_sequence_i` from the common parts?

Reply:
Factored out gen_array and gen_sequence; removed gen_anonymous_type.